packages/google_maps_flutter/google_maps_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.13.0
+
+* Adds support for disabling or moving the camera control button on web.
```

Comment:
This change should be made to all the changelogs, not just this one.

Reply:
@stuartmorgan-g i updated google_maps_flutter, google_maps_platform_interface & google_maps_flutter_web. Android & iOS I didn't. Is it right?